modules/tensor_mechanics/src/materials/ADComputeMultipleInelasticStress.C

```diff
@@ -390,6 +390,9 @@ ADComputeMultipleInelasticStress::computeAdmissibleState(
     ADRankTwoTensor & elastic_strain_increment,
     ADRankTwoTensor & inelastic_strain_increment)
 {
+  // Properly update material properties (necessary if substepping is employed).
+  _models[model_number]->zeroOutIncrementalMaterialProperties();
```

Comment:
Do we have the mechanism to prevent calling this if sub-stepping is not enabled for one of the models? 

The users may get into cases when one/several of the multiple inelastic models do not support sub-stepping.

Reply:
This would become an empty call up in the hierarchy if the model doesn't support substepping. Calling it should be safe for any model.